trace_viewer/core/location.html

```diff
@@ -67,11 +87,33 @@
           this.yComponents_.push(
               new YComponent(elem.eventContainer.stableId, yPercentOffset));
         }
-
         elem = elem.parentElement;
       }
     },
 
+    fromStableIdAndTs: function(stableId, ts) {
```

Comment:
nit: Make this fromStableIdAndTimestamp  (My initial though was, what are the T's?)